DisguiseUnityRenderStream/Runtime/PluginEntry.cs

```diff
@@ -780,6 +784,7 @@ private PluginEntry()
                 Debug.LogError(string.Format("Failed to initialise: {0}", error));
             else
             {
+#if !UNITY_EDITOR
```

Comment:
The graphics init was executing during build and failing, in turn preventing the schema from being written out. I must have introduced the bug a few PRs back.

Reply:
This means that the PluginEntry() static ctor is the wrong place for calling our graphics init.